gocwebtemplate-sample-jsp/src/main/resources/goc/webtemplate/global/config/cdn.properties

```diff
@@ -29,7 +29,7 @@ cdn_localdev_localpath=/GoCWebTemplateSampleJSP/loadtesting/app/cls/WET/{0}/{1}/
 #(blank values for theme, subtheme and version will use defaults from internal environment configuration)
 wettemplate_theme=
 wettemplate_subtheme=
-wettemplate_version=v4_0_45
+wettemplate_version=v4_0_46

```

Comment:
Try with blank to see if JSP sample still works fine, if it does let's remove it.

Reply:
Tested with blank and it works perfectly.